File: RB100/age.rb
def age_increments(age, decades)
	(decades+1).times do |x|
		next if x == 0
		puts "In #{x*10} years you will be #{age+(x*10)} years old!"
	end
end


puts "How old are you?"
age = gets.chomp.to_i

age_increments(age, 4)

File: JS210/intro_book/intro_prog_ex/io/age.js
let rlSync = require('readline-sync');
let startingAge = parseInt(Number(rlSync.question('How old are you currently?\n')));
agesImpending = [10, 20, 30, 40];
agesImpending.forEach(ageAddition => console.log(`In ${ageAddition} years, you will be ${startingAge + ageAddition} years old.`));
